### ISSUE TITLE
Problem with MD on MAC OS X El Capitan

### ISSUE THREAD

**Issue by HeyItsAnTs** [atomizer/muledump#126]
Hey, I have been getting this annoying blank whenever I open the muledump.html file. I am sure I entered the correct login details. I've tried all browsers as well. I also tried opening the accounts javascript file with the browser to provide more information. Please help! Thanks! 

<img width="1440" alt="screen shot 2017-02-13 at 9 07 55 pm" src="https://cloud.githubusercontent.com/assets/25758649/22911933/8e2c1a20-f230-11e6-8276-e114aad130d7.png">

<img width="584" alt="screen shot 2017-02-13 at 9 03 56 pm" src="https://cloud.githubusercontent.com/assets/25758649/22912015/e9b9386e-f230-11e6-96bf-3acf7ca1e427.png">

**Comment by Nightfirecat** [comment]
Can you please open the browser console (search for the shortcut to open it if you don't know how to access it), reload the page, then post what the error message there says?

**Comment by HeyItsAnTs** [comment]
Hey are you talking about this? 


<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8">
	<title>Muledump</title>
	<script type="application/javascript" src="vendor/jquery.min.js"></script>
	<script type="application/javascript" src="vendor/masonry.pkgd.min.js"></script>
	<script type="application/javascript" src="lib/constants.js"></script>
	<script type="application/javascript" src="lib/sheets.js"></script>
	<script type="application/javascript" src="lib/portrait.js"></script>
	<script type="application/javascript" src="lib/pcstats.js"></script>
	<script type="application/javascript" src="lib/export.js"></script>
	<script type="application/javascript" src="accounts.js"></script>
	<script type="application/javascript" src="lib/realmapi.js"></script>
	<script type="application/javascript" src="lib/options.js"></script>
	<script type="application/javascript" src="lib/totals.js"></script>
	<script type="application/javascript" src="lib/mule.js"></script>
	<script type="application/javascript" src="lib/muledump.js"></script>
	<link type="text/css" rel="stylesheet" href="lib/dump.css">
	<link type="image/png" rel="shortcut icon" href="lib/favicon.png">
</head>
<body>
	<img id="renders" alt="renders" src="lib/renders.png" style="display:none">
	<span id="accopts" style="display: none;" class="menu"></span>
	<div id="main">
		<div id="top">
			<div id="cnt" style="display: none;">waiting requests: <span id="counter"></span></div>
			<div class="handle"><span>options</span><span id="options" class="menu"></span></div>
			<div id="reloader">reload all</div>
			<div class="handle"><span>export</span><span id="export" class="menu"></span></div>
			<div id="update">update</div>
			<a href="https://github.com/atomizer/muledump" target="_blank">home</a>
		</div>
		<div id="totals"></div>
		<div id="errors"></div>
		<div id="stage"></div>
	</div>
</body>
</html>

**Comment by Nightfirecat** [comment]
No, that is the page's HTML source. [Please follow this answer for opening the developer console](http://webmasters.stackexchange.com/questions/8525/how-to-open-the-javascript-console-in-different-browsers#77337), then open the "Error console". It should looks something like this:

![Safari JavaScript error console](http://cdn.inmotionhosting.com/support/images/stories/website/how-to/diagnose-error-console/javascript-problems-12-safari-error-console.gif)

**Comment by HeyItsAnTs** [comment]
Alright here.

**Comment by HeyItsAnTs** [comment]
![screen shot 2017-02-14 at 1 16 44 pm](https://cloud.githubusercontent.com/assets/25758649/22943311/1b7c2520-f2bb-11e6-9345-fa8702e32469.png)

**Comment by HeyItsAnTs** [comment]
Does this work?

**Comment by Nightfirecat** [comment]
Yes, that's excellent--the error line saying `Uncaught SyntaxError` means that you've put in some kind of invalid formatting in your accounts.js file.

Can you please replace your usernames/passwords in the file with dummy values, then copy+paste the contents of that file here? Alternately, just make sure that you're using regular single quotes `'` or double quotes `"` instead of fancy single `‘’` and double `“”` quotes.

**Comment by HeyItsAnTs** [comment]
I'm not sure where you're getting at. I made sure I"m not using double quotes already -_-

**Comment by Nightfirecat** [comment]
Well, something else about the formatting of the file is incorrect; I can't tell you what just from the error message. (Javascript's parsing error is always very generic)
